### PR TITLE
Fix #430 add pkjson.dump_str and log pkunit case_dir

### LINKED ISSUE #430: add pkjson.dump and pkunit.case_dirs print case

### PR DESCRIPTION
**Pull request by robnagler**
- Fix github_test to use new structure of github._TEST_REPOS

### CODE REVIEW

File: pykern/pkjson.py [hunk 1]
```diff
@@ -60,16 +72,16 @@ def dump_pretty(obj, filename=None, pretty=True, **kwargs):
...
 
     Args:
         obj (object): any Python object
         kwargs (object): other arguments to `dump_pretty`
     Returns:
-        bytes: (unsorted) formatted JSON
+        str: (unsorted) formatted JSON
     """
-    return dump_pretty(obj, pretty=False, **kwargs).encode(ENCODING)
+    return dump_pretty(obj, pretty=False, **kwargs)
```
Comment (Benbenbenin0) [inline pykern/pkjson.py:84]: What is dump_str() for? Is it also for network transfers or just a generally useful conversion?

File: tests/pkcli/github_test.py [hunk 2]
```diff
@@ -177,4 +176,5 @@ def _create_commit(repo, full_name=False):
 def _test_repo():
     from pykern.pkcli import github
 
-    return "/".join(github._TEST_REPOS[0])
+    t = github._TEST_REPOS[0]
```
Comment (Benbenbenin0) [inline tests/pkcli/github_test.py:179]: Was this broken before or just a rewrite?

**Review by Benbenbenin0 — APPROVED** [review]
Looks good, had some clarifying questions.

File: pykern/pkjson.py (hunk 1 above)
Comment (Benbenbenin0) [reply pykern/pkjson.py:84]: Nvm, covered it in the software meeting

File: tests/pkcli/github_test.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```
[the remark this answers is not in the document]
Comment (robnagler) [reply tests/pkcli/github_test.py:1]: It isn't necessary any more. It should get [removed from home-env](https://github.com/biviosoftware/home-env/issues/102).

**Pull request merged**